src/services/api.ts

```diff
@@ -130,7 +130,14 @@ const register = async (username: string, password: string, email: string): Prom
 
 const resetPassword = async (email: string): Promise<{}> => {
   const data = { email }
-  const url = BACKEND_URL + '/account/reset/password'
+  const url = BACKEND_URL + '/auth/users/reset_password'
```

Comment:
Missing a trailing slash, leading to a 302 from the backend then a CORS error

Reply:
resolved